### DISCUSSION TITLE
更新后node无法被正常启动

### DISCUSSION THREAD

**Discussion by ArimaKana-Soda** [Arondight/Adachi-BOT#766] [Q&A] [answered]
原本是可以正常运行的，但是今天尝试更新后出现以下错误[](url
![Snipaste_2022-03-30_11-54-06](https://user-images.githubusercontent.com/88731781/160748208-dbb3f8b8-8f88-43ee-93e7-c7d7171f9189.png)
)
任务管理器中也没发现node被正常启动
尝试npm install 再npm run init也是出现这种情况

**Comment by mark9804** [comment]
#140 , https://github.com/Arondight/Adachi-BOT/issues/760#issuecomment-1081916202

看一下 node 安装路径在不在 PATH 变量里面，然后看一下是不是 npm 版本过高

**Reply by Arondight** [reply]
干脆我们把 node 也管理起来吧，其实这样对我们也有好处

**Comment by Arondight** [comment]
```sh
git pull -p
npm install
npm run init
```

**Reply by ArimaKana-Soda** [reply]
![Snipaste_2022-03-30_13-44-33](https://user-images.githubusercontent.com/88731781/160759801-2f4d2ad7-ff74-4a73-99de-91149d39a072.png)
请问是不是因为不是这样用的
已经又下载了最新版
现在结果还是这样
![Snipaste_2022-03-30_13-44-46](https://user-images.githubusercontent.com/88731781/160759869-dd549646-38cf-4755-b9ba-51fcaa9827cc.png)

**Reply by Arondight** [reply]
```
git pull orign master -p
```

**Reply by ArimaKana-Soda** [reply]
![image](https://user-images.githubusercontent.com/88731781/160765977-87acfdbb-9477-456d-b4d6-e268f0bde225.png)
可是这样诶
如果是更新的话，我现在下载的就是一小时前更新的最新版的

**Reply by Arondight** [reply]
```
git pull https://github.com/Arondight/Adachi-BOT.git master -p
```

**Reply by Arondight** [reply]
总之能用就行了，别纠结了

**Reply by ArimaKana-Soda** [reply]
前面三步都执行了，但是但是我不能用呀QAQ
![image](https://user-images.githubusercontent.com/88731781/160768478-5851d8b3-db47-4ecb-a3e0-c5a48173a44b.png)
现在是不可以登入的，之前一版的可以

**Reply by Arondight** [reply]
```
git pull https://github.com/Arondight/Adachi-BOT.git master -p
npm install
npm run init
```

**Reply by ArimaKana-Soda** [reply]
是这样子做的
![image](https://user-images.githubusercontent.com/88731781/160769036-5a7bcc5e-2cb4-4afa-b9d6-5233d2922748.png)
![image](https://user-images.githubusercontent.com/88731781/160769088-c9fa3345-4e92-4e05-bde2-e31170b597a3.png)
![image](https://user-images.githubusercontent.com/88731781/160769109-f31d0bf5-80f7-41d1-a660-158b6f4c9f67.png)

**Reply by Arondight** [reply]
```
cat .git/config
```

**Reply by ArimaKana-Soda** [reply]
![image](https://user-images.githubusercontent.com/88731781/160770416-a323bda1-c88f-4a42-9e2b-9d56a51f348f.png)
这样子，再怎么做呢

**Reply by Arondight** [reply]
我回滚一下提交，本地重新配置 node 吧，npm 不提供平台统一的 node 模块

**Reply by Arondight** [reply]
```
npm remove node
```

或者再更新一下代码，然后配置一下 windows 环境变量，可以参考 #140

**Comment by Arondight** [comment]
npm 更新后确实有问题，你现在 package.json 里面，把所有的 npx 都删掉

**Comment by Arondight** [comment, marked as answer]
再更新一下应该可以了， npx 的行为变了

**Reply by ArimaKana-Soda** [reply]
嗯嗯解决了
